components/operator/config/rbac/role.yaml

```diff
@@ -132,3 +132,15 @@ rules:
   - get
   - list
   - watch
+- apiGroups:
```

Comment:
Can be removed (not a blocker).

Reply:
I prefer to keep it I think I will need it in the future